go.mod

```diff
@@ -125,4 +126,6 @@ require (
 	gopkg.in/spacemonkeygo/monkit.v2 v2.0.0-20180827161543-6ebf5a752f9b
 	gopkg.in/vmihailenco/msgpack.v2 v2.9.1 // indirect
 	gopkg.in/yaml.v2 v2.2.2 // indirect
+	labix.org/v2/mgo v0.0.0-20140701140051-000000000287 // indirect
+	launchpad.net/gocheck v0.0.0-20140225173054-000000000087 // indirect
```

Comment:
Where did gocheck, mgo and appengine come from?

Reply:
I'm honestly not sure

Reply:
Looks like `go mod tidy` is adding these